### PR TITLE
feat(cli): add command to migrate git files to lfs

### LINKED ISSUE #1250: A workflow to fix broken repositories due to large files in Git history
As a Renku CLI user, I want to have a workflow that guides me how to fix a broken repository that contains large files in its history.

**Describe the solution you'd like**
- A command that migrates files to Git LFS and rewrites the history (see `git lfs migrate import`)
- All Renku-related metadata that use a commit hash should be updated with new commit hashes.
- Remote repo should be force-pushed.

**Describe alternatives you've considered**
A document that shows how to do the above steps.

**Additional context**
Related: https://github.com/SwissDataScienceCenter/renku-python/issues/467

### PR DESCRIPTION
**Pull request by Panaetius**
Adds `renku storage fix` command that migrates objects to git lfs

closes #1250

### CODE REVIEW

**Comment by rokroskar** [comment]
This is great! I'm wondering if the command should be a bit more conservative, i.e. require a specific path or `--all`. It's a destructive operation that will be impossible for a normal user to undo, unless they are friends of `git reflog`

File: tests/core/commands/test_storage.py
Comment (jsam) [inline tests/core/commands/test_storage.py:104]: Just a small suggestion.

Since, `client.path` is `Path` object, you could simplify this by saying 

```
(client.path / "dataset_file").write_text("dataset file")
```

then all those file creation could be something like

```
for _file in ["dataset_file", "workflow_file", "regular_file"]:
    (client.path / _file).write_text(_file.replace("_", " "))
```

where, `.replace` could be omitted.

File: tests/core/commands/test_storage.py
Comment (jsam) [inline tests/core/commands/test_storage.py:144]: Analogous to above comment, there is also `.read_text()` on `Path` object, which could be utilized here.

**Review by jsam — COMMENTED** [review]
Thanks! `renku storage migrate` is much more lean and clean :sunglasses: 

Look's really great. I added small suggestion, let me know if you want to implement it. Otherwise we can :rocket: it!

**Comment by rokroskar** [comment]
Thanks @Panaetius lgtm now from a usability PoV - I'll let @jsam approve once he's happy with the code

**Review by jsam — APPROVED** [review]
🚀 🚀 🚀 

🚢 it!

**Pull request merged**
